Make MOOSE use latest moose-environment

milljm opened the pull request:
This PR will instruct Civet to test against the recent package_builder merge.
Update download package links on mooseframework.org.

Note: An additional change to civet recipes is required to use python3 instead
of python2.

Closes #14255

Commit:
Make MOOSE use latest moose-environment

This PR will instruct Civet to test against the recent package_builder merge.
Update download package links on mooseframework.org.

Closes #14255

moosebuild commented:
All [jobs](https://civet.inl.gov/event/47909/) on b344c5b : invalidated by @permcody

CIVET pause

Commit:
moose-environment, package_version update

Add support for Catalina.
Update to GCC 9.2/LLVM 9.
Update to Modules 4.3.1.
Add VTK-Mesa support (offscreen rendering).

moosebuild commented:
Job [Test timings](https://civet.inl.gov/job/422574/) on 1e74c30 wanted to post the following:

View timings [here](https://mooseframework.inl.gov/docs/PRs/14256/timing)

This comment will be updated on new commits.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/422531/) on 1e74c30 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/14256/site)

This comment will be updated on new commits.

aeslaughter committed:
Add 'display_required' to chigger testing

aeslaughter committed:
Update sqa_check for missing import

milljm commented:
A fix to code coverage will require a new package_version.

Commit:
Trigger new build, LCOV has been updated

Updated package_version to test new LCOV version (fix code coverage).
Added OpenSUSE back into the mix.

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/424745/) on 2aef403 wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/14256/coverage)

Commit:
Decode subprocess output

Fix the joining of bytes like object.

Commit:
Add that pesky comma

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/425347/) on a272740 wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/14256/coverage)

moosebuild commented:
Job [Test Parallel debug](https://civet.inl.gov/job/425360/) on a272740 : invalidated by @permcody

All hell broke loose

permcody commented:
Hold onto your butts!
